web-app/js/portal/details/NcWmsPanel.js

```diff
@@ -39,10 +39,16 @@ Portal.details.NcWmsPanel = Ext.extend(Ext.Container, {
 
         this._disableDateTimeFields();
         this._attachTemporalEvents();
+
+        if (this.layer.temporalextentloaded) {
```

Comment:
Or, if it is information about the layer and loading - perhaps this code should be moved to the layer or to wherever the loading occurs.  It just seems odd updating one from the other here.